backend/src/main/java/com/gestionnaire_de_stage/exception/EmailDoesNotExistException.java

```diff
@@ -1,4 +1,17 @@
 package com.gestionnaire_de_stage.exception;
 
+import lombok.Getter;
+
+@Getter
 public class EmailDoesNotExistException extends Exception {
+    private final String message;
+
+    public EmailDoesNotExistException() {
+        this.message = "L'email n'existe pas";
+    }
+
+    public EmailDoesNotExistException(String message) {
+        this.message = message;
+    }
+
```

Comment:
```suggestion

public class EmailDoesNotExistException extends Exception {
    public EmailDoesNotExistException(String message) {
        super(message);
    }
    public EmailDoesNotExistException() {
        super("L'email n'existe pas");
    }
    

```

On set jamais un message avec un setter dans une exception, on fais juste instancié une nouvelle exception.
Et le message dois être envoyer normalement lors de l'instanciation.
T'as aussi pas besoin de `@Getter` puisque la classe parent te donne accès au message à travers `getMessage()`